sky/clouds/service_catalog/data_fetchers/fetch_fluidstack.py

```diff
@@ -0,0 +1,171 @@
+        new_plan['configuration']['nvme_storage'] = config['nvme_storage']
+        new_plan['plan_id'] = f'custom:{i}:{plan["plan_id"]}'
+        new_plans.append(new_plan)
+    return new_plans
+
+
+def create_catalog(output_dir: str) -> None:
+    response = requests.get(ENDPOINT)
+    plans = response.json()
+    custom_plans = [
+        plan for plan in plans if plan['minimum_commitment'] == 'hourly' and
+        plan['type'] in ['custom'] and plan['gpu_type'] != 'NO GPU'
+    ]
+    #plans = [plan for plan in plans if len(plan['regions']) > 0]
+    plans = [
+        plan for plan in plans if plan['minimum_commitment'] == 'hourly' and
+        plan['type'] in ['preconfigured'] and
+        plan['gpu_type'] not in ['NO GPU', 'RTX_3080_10GB', 'RTX_3090_24GB']
+    ]
+
+    plans = plans + [
+        plan for plan in custom_plans for plan in plans_from_custom_plan(plan)
+    ]
+    with open(os.path.join(output_dir, 'vms.csv'), mode='w') as f:
+        writer = csv.writer(f, delimiter=',', quotechar='"')
+        writer.writerow([
+            'InstanceType',
+            'AcceleratorName',
+            'AcceleratorCount',
+            'vCPUs',
+            'MemoryGiB',
+            'Price',
+            'Region',
+            'GpuInfo',
+            'SpotPrice',
+        ])
+        for plan in plans:
+            try:
+                gpu = GPU_MAP[plan['gpu_type']]
+            except KeyError:
+                print(f'Could not map {plan["gpu_type"]}')
+                continue
+            gpu_memory = int(
+                str(plan['configuration']['gpu_memory']).replace('GB',
+                                                                 '')) * 1024
+            gpu_cnt = int(plan['configuration']['gpu_count'])
+            vcpus = float(plan['configuration']['core_count'])
+            mem = float(plan['configuration']['ram'])
+            price = float(plan['price']['hourly']) * gpu_cnt
+            gpuinfo = {
+                'Gpus': [{
+                    'Name': gpu,
+                    'Manufacturer': 'NVIDIA',
+                    'Count': gpu_cnt,
+                    'MemoryInfo': {
+                        'SizeInMiB': int(gpu_memory)
+                    },
+                }],
+                'TotalGpuMemoryInMiB': int(gpu_memory * gpu_cnt),
+            }
+            gpuinfo = json.dumps(gpuinfo).replace('"', "'")  # pylint: disable=invalid-string-quote
+            for r in plan.get('regions', []):
+                if r['id'] == 'india_2':
+                    continue
+                writer.writerow([
+                    plan['plan_id'],
+                    gpu,
+                    gpu_cnt,
+                    vcpus,
+                    mem,
+                    price,
+                    r['id'],
+                    gpuinfo,
+                    '',
+                ])
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('catalog_dir', required=False)
```

Comment:
`required` is not a valid argument for positional. How about we just keep the logic the same as our other clouds, i.e., saving to the current directory